### PR TITLE
Fix goreleaser job

### PR DESCRIPTION
**Pull request by nicoelzer**
## Description

Updated the Github Workflow for the release job by providing the `workdir` argument which is now pointing to the correct folder, which is required as the main function is not located in the root folder as expected by default. Reference: https://github.com/goreleaser/goreleaser-action#customizing

closes: #310

### CODE REVIEW

**Review by adlerjohn — COMMENTED** [review]
Merge conflicts

**Comment by nicoelzer** [comment]
> Merge conflicts

Just resolved them @adlerjohn

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -22,5 +22,6 @@ jobs:
         uses: goreleaser/goreleaser-action@v2.9.1
         with:
           args: release --rm-dist
+          workdir: ./cmd/celestia-appd
```
Comment (adlerjohn) [inline .github/workflows/tag.yml:25]: Shouldn't this be `./cmd/`, since `celestia-appd` is a file and not a directory?

Comment (evan-forbes) [reply .github/workflows/tag.yml:25]: @nicoelzer bump, I have the same question

Comment (adlerjohn) [reply .github/workflows/tag.yml:25]: This actually sets it to `./cmd/celestia-appd/` (the directory)!

**Review by adlerjohn — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**